docs/index.md

```diff
@@ -10,8 +10,15 @@ application with the full power of ArcGIS.
 
 ## Documentation Pages
 - [Getting Started](pages/gettingStarted) - How to set up your first GeoBlazor application
+- [Features](pages/features) - How to set up your first GeoBlazor application
```

Comment:
I don't think this description is right. The `Features` page is just a list of features. The features list needs a different description.

Reply:
That was strange, I guess I had not saved prior to committing.  
It is corrected.  This is the description: "Available features supported by GeoBlazor"